api/src/main/java/jakarta/data/metamodel/Attribute.java

```diff
@@ -15,16 +15,14 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-package jakarta.data.model;
+package jakarta.data.metamodel;
 
 import jakarta.data.Sort;
 
 /**
- * Implemented by the Jakarta Data provider to
- * {@link Attribute#init(AttributeInfo) initialize} an attribute field
- * in the {@link StaticMetamodel}.
+ * Represents an attribute of a Java type.
  */
-public interface AttributeInfo {
+public interface Attribute {
```

Comment:
Elsewhere in the pull you refer to 
`{@link Attribute#init(Attribute) initialize}`
but you have not included an `init` method on `Attribute`.